### ISSUE TITLE
Metallic artifacts in SiFiGAN

### ISSUE THREAD

**Issue by ChuhanWang10** [chomeyama/SiFiGAN#15]
Hi,

Thanks for the interesting work! I have tried to train this SiFiGAN vocoder with LJspeech corpus. I resampled the audio data to 24kHz and normalized them to -26db according to the data preparation part in the paper. I used Univnet discriminator, and the rest of the hyperparameter settings I barely did not change. However, after 500k steps training, there are some metallic sounds in the analysis-resynthesis results (prosody and other stuff are okay). Could you share some tricks about how to avoid these artifacts when training the model? 

Thank you!
